blob/service.go

```diff
@@ -354,14 +354,153 @@ func (s *Service) retrieve(
 		return nil, nil, err
 	}
 
-	eds, err := s.shareGetter.GetEDS(ctx, header)
+	headerGetterSpan.SetStatus(codes.Ok, "")
```

Comment:
tracing: Ok status should be set once on success. 

Currently, it's written always by default, while error cases do not overwrite it, so error op may show as success.

Reply:
Actually, it is correct. I missed the fact that there several spans in this method, which we agreed not to refactor in this PR with @rach-id

Reply:
upon sync discussion, this is how it was previously, and a separate issue would take care of hiding the statuses in their corresponding methods.